src/fava/application.py

```diff
@@ -270,11 +270,19 @@ def fava_api_exception(error: FavaAPIException):
 
 @app.route("/")
 @app.route("/<bfile>/")
-def index():
+def index(bfile=None):
     """Redirect to the Income Statement (of the given or first file)."""
     if not g.beancount_file_slug:
         g.beancount_file_slug = next(iter(app.config["LEDGERS"]))
-    return redirect(url_for("report", report_name="income_statement"))
+    default_path = ""
+    if not bfile:
+        default_path += g.beancount_file_slug + "/"
+    default_path += (
+        app.config["LEDGERS"][g.beancount_file_slug]
+        .fava_options["default-path"]
+        .lstrip("/")
+    )
+    return redirect(f"/{default_path}")
```

Comment:
This can't be an absolute path at the root but rather needs to respect the path that WSGI app is mounted at.

Reply:
I think I understand what you're saying in terms of needing to be a relative path, but am not entirely sure I understand how to translate that into the implementation here. Is it as simple as not appending the leading "/" on this line?

Reply:
I think it would be better to keep this as an absolute path, but you'll need to use Flask to figure out what the path prefix is, `url_for('index')` might work.

Reply:
Okay, `url_for('index')` does appear to be working, as you suggested. Thanks!